dianna/methods/lime.py

```diff
@@ -276,3 +282,27 @@ def moveaxis_function(data):
         if self.preprocess_function is None:
             return moveaxis_function
         return lambda data: self.preprocess_function(moveaxis_function(data))
+
+    @staticmethod
+    def get_explanation_values(label: int, explanation: ImageExplanation) -> NDArray:
```

Comment:
```suggestion
    def _get_explanation_values(self, label: int, explanation: ImageExplanation) -> np.array:
```
Make it a hidden function.